contracts/MMRStatelessVerification.sol

```diff
@@ -0,0 +1,50 @@
+// "SPDX-License-Identifier: UNLICENSED"
+pragma solidity ^0.7.0;
+
+contract MMRStatelessVerification {
+
+    // Verifies is a parachain header is contained in an MMR leaf
+    function verifyParachainHeaderInLeaf(
+        bytes32 paraHead,
+        bytes32 paraHeadSibling,
+        bytes32 leafParaHeadRoot
+    ) public pure returns (bool) {
+        return leafParaHeadRoot == keccak256(abi.encodePacked(paraHead, paraHeadSibling));
```

Comment:
The order of the headers should matter here, I guess we need to either sort head and head sibling or submit the order via a boolean (depending on how the headers are ordered in the leaf).

Reply:
Updated in https://github.com/Snowfork/verification-solidity/pull/6/commits/d79265aaa79dac6a8c0c823120c4ddbe141ed2f7

Reply:
Looks good to me 👍